Add fergusstrange/embedded-postgres

fergusstrange opened the pull request:
Hi,

All requested metrics / documentation as follows...

- github.com repo: https://github.com/fergusstrange/embedded-postgres
- godoc.org: https://godoc.org/github.com/fergusstrange/embedded-postgres
- goreportcard.com: https://goreportcard.com/report/github.com/fergusstrange/embedded-postgres
- coverage service link: https://coveralls.io/github/fergusstrange/embedded-postgres?branch=master

Thanks!

avelino commented:
Thank you for contributing with [awesome-go](https://github.com/avelino/awesome-go), we will revise your contribution as soon as possible.

Automation body content check:
- godoc.org: True
- goreportcard.com: True
- coverage: True

fergusstrange commented:
@panjf2000 I've resolved the golint issues and below is the full template (I'd reused the content of an old pull request that was accepted before).

Please check if what you want to add to `awesome-go` list meets [quality standards](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard) before sending pull request. Thanks!

**Please provide package links to:**

- github.com repo: https://github.com/fergusstrange/embedded-postgres
- godoc.org: https://godoc.org/github.com/fergusstrange/embedded-postgres
- goreportcard.com: https://goreportcard.com/report/github.com/fergusstrange/embedded-postgres
- coverage service link: https://coveralls.io/github/fergusstrange/embedded-postgres?branch=master

Very good coverage

**Note**: that new categories can be added only when there are 3 packages or more.

**Make sure that you've checked the boxes below before you submit PR:**
- [ ] I have added my package in alphabetical order.
- [ ] I have an appropriate description with correct grammar.
- [ ] I know that this package was not listed before.
- [ ] I have added godoc link to the repo and to my pull request.
- [ ] I have added coverage service link to the repo and to my pull request.
- [ ] I have added goreportcard link to the repo and to my pull request.
- [ ] I have read [Contribution guidelines](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#contribution-guidelines), [maintainers note](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#maintainers) and [Quality standard](https://github.com/avelino/awesome-go/blob/master/CONTRIBUTING.md#quality-standard).

Thanks for your PR, you're awesome! :+1:
